templates/includes/records/hierarchy-global.html

```diff
@@ -1,29 +1,236 @@
+
+                        {% elif forloop.counter == 3 and not is_current_item %}
+                            <li class="hierarchy-short-panel__list-item">
+                                <span class="hierarchy-short-panel__level">
+                                    <a class="hierarchy-short-panel__link" href="{% record_url item %}">Piece</a>
+                                </span>({{ item.reference_number }})
+                            </li>
+
+                        {% elif is_current_item %}
+                            <li class="hierarchy-short-panel__list-item hierarchy--short-panel__list-item--current">
+                                <span class="hierarchy-short-panel__level">This {{ page.level|lower }}</span>({{ item.reference_number }})
+                            </li>
+                        {% endif %}
+                    {% endfor %}
                 {% else %}
-                    <li class="hierarchy-global__list-item">
-                        <span class="hierarchy-global__reference">{{ item.reference_number }}</span>
-                        <a href="{% record_url item %}"  data-link="{{ forloop.counter0 }}: {{ item.reference_number }}" class="link--bold">
-                            {{ item.title }}
-                        </a>
+                    <li class="hierarchy-short-panel__list-item">
+                        <span class="hierarchy-short-panel__level">
+                            <a href="#">Archive</a>
+                        </span>({{ page.template.heldBy }})
+                    </li>
+                    <li class="hierarchy-short-panel__list-item hierarchy--short-panel__list-item--current">
+                        <span class="hierarchy-short-panel__level">This department</span>({{ page.reference_number }})
                     </li>
                 {% endif %}
-            {% endfor %}
-        </ol>
-    </details>
-</nav>
+            </ul>
+
+        </nav>
+
+        <div class="hierarchy-short-panel__toggle-link" data-id="hierarchy-toggle-link"></div>
+
+        <!-- DETAILED HIERARCHY VIEW -->
+        <nav class="hierarchy-full-panel" role="navigation" data-id="hierarchy-detailed-view">
+            <h2 class="hierarchy-full-panel__heading">Detailed catalogue location:</h2>
+
+            <!-- Level 1 -->
+            <div class="hierarchy-full-panel__bar-container-outer">
+                <div class="hierarchy-full-panel__bar-container-inner hierarchy-full-panel__bar-container-inner--level-1">
+                    <p><span class="hierarchy-full-panel__bar-number">N</span>
+                    <span class="hierarchy-full-panel__bar-label">records</span></p>
+                </div>
+                <div class="hierarchy-full-panel__text-container">
+                    <p class="hierarchy-full-panel__text-level">This record is held at: <a href="">{{page.template.heldBy}}</a></p>
+                    <p class="hierarchy-full-panel__text-title">Located at DATA TO GO HERE</p>
```

Comment:
Could we put a TODO comment here above line containing `Located at DATA TO GO HERE`